components/Post/Post.js

```diff
@@ -114,7 +113,22 @@ export function Post({ post }) {
   }
 }
 
-export const Container = styled.article`
+const DeleteIcon = styled(RiDeleteBin2Line)`
```

Comment:
the name `RiDeleteBin2Line` seems odd to me, is it possible to rename it to something more meaningful?

Reply:
hey, that's an imported icon vom react